.github/workflows/test-image-execution.yaml

```diff
@@ -0,0 +1,123 @@
+name: Test Image execution on OS
+
+on:
+  # run it on push to the default repository branch
+  push:
+    branches: [master]
+  pull_request:
+  # Only trigger, when the build workflow succeeded
+  # workflow_run:
+  #   workflows: [ "Deploy Application images when Merging on Master" ]
+  #   types: [completed]
+
+env:
+  LISTINGAPI_ENDPOINT: "http://localhost:8181/listings/32682"
+  PRICEMAP_PYTHON_ENDPOINT: "http://localhost:8080"
+  PRICEMAP_TYPESCRIPT_ENDPOINT: "http://localhost:8080"
+
+jobs:
+  build:
+    name: Test containers execution
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+    services:
+      db:
+        image: "mdillon/postgis:11-alpine"
+        env:
+          POSTGRES_DB: pricemap
+          POSTGRES_USER: pricemap
+          POSTGRES_PASSWORD: pricemap
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 20s
+          --health-timeout 5s
+          --health-retries 5
+          --health-start-period 15s
+        ports:
+          # Maps tcp port 5432 on service container to the host
```

Comment:
It the opposite: it maps the port 5432 on the host to the port 5432 on the container.